examples/server/app/dash/main.py

```diff
@@ -2,10 +2,9 @@
 This example demonstrates the use of Bootstrap templates to compile several types
 of charts in one file.
 
-.. note::
+note:
     This example needs the Pandas package to run.
-
-.. _Auto MPG: https://archive.ics.uci.edu/ml/datasets/auto+mpg
+    _Auto MPG: https://archive.ics.uci.edu/ml/datasets/auto+mpg
```

Comment:
@PabloRuizCuevas All the changes to the docstring need to be reverted. The removed bits are Sphinx directives and are there intentionally.

Reply:
@bryevdv ups true, now is fixed, thanks for the feedback.